.github/workflows/release-sims.yml

```
@@ -14,11 +12,11 @@ jobs:
       - uses: rokroskar/workflow-run-cleanup-action@master
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
-    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main'"
+    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main' && github.event.label.name == 'release'"
```

Then, the condition should be modified as below:
```suggestion
    if: "!startsWith(github.ref, 'refs/tags/') && github.ref != 'refs/heads/main' && contains(github.event.pull_request.labels.*.name, 'release')"
```